engine/apps/mattermost/client.py

```diff
@@ -0,0 +1,70 @@
+from dataclasses import dataclass
+from typing import Optional
+
+import requests
+from requests.auth import AuthBase
+from requests.models import PreparedRequest
+
+from apps.base.utils import live_settings
```

Comment:
```suggestion
```
We are considering to limit/deprecate the use of `live_settings` (preferring Django settings + env vars)